ee/tabby-ui/app/(dashboard)/settings/(integrations)/gitops/detail/components/detail.tsx

```diff
@@ -0,0 +1,266 @@
+          </div>
+          <div className="flex items-center gap-2 text-base">
+            <IconGitHub className="h-6 w-6" />
+            GitHub.com
+            <div className="ml-1">
+              {provider?.connected ? (
+                <Badge variant="successful">Connected</Badge>
+              ) : (
+                <Badge variant="destructive">Not Connected</Badge>
+              )}
+            </div>
+          </div>
+        </CardTitle>
+      </CardHeader>
+      <CardContent className="pl-0">
+        <LoadingWrapper loading={fetching} fallback={<ListSkeleton />}>
+          <UpdateProviderForm
+            defaultValues={provider}
+            onDelete={onDeleteProvider}
+            onBack={() => {
+              router.push('/settings/gitops')
+            }}
+            id={id}
+          />
+        </LoadingWrapper>
+      </CardContent>
+
+      <CardHeader className="mt-8 pl-0 pt-0">
+        <CardTitle className="flex items-center justify-between">
+          <span>Linked repositories</span>
+          <Dialog open={open} onOpenChange={setOpen}>
+            <DialogContent>
+              <DialogHeader className="gap-3">
+                <DialogTitle>New repository link</DialogTitle>
+                <DialogDescription>
+                  Add a new repository link to this provider.
+                </DialogDescription>
+              </DialogHeader>
+              <LinkRepositoryForm
+                onCancel={() => setOpen(false)}
+                onCreated={onCreated}
+                repositories={unlinkedRepos}
+                fetchingRepositories={fetchingRepositories}
+              />
+            </DialogContent>
+            <DialogTrigger asChild>
+              <Button>New repository link</Button>
+            </DialogTrigger>
+          </Dialog>
+        </CardTitle>
+      </CardHeader>
+      <CardContent className="pl-0">
+        <LoadingWrapper loading={fetchingRepositories}>
+          <LinkedRepoTable data={linkedRepos} />
+        </LoadingWrapper>
+      </CardContent>
+    </LoadingWrapper>
+  )
+}
+
+const LinkedRepoTable: React.FC<{
+  data: GithubRepositories | undefined
+  onDelete?: () => void
+}> = ({ data, onDelete }) => {
+  // const [isDeleting, setIsDeleting] = useState(false)
+  const updateGithubProvidedRepositoryActive = useMutation(
+    updateGithubProvidedRepositoryActiveMutation,
+    {
+      onCompleted(data) {
+        if (data?.updateGithubProvidedRepositoryActive) {
+          onDelete?.()
+        }
+        // setIsDeleting(false)
+      },
+      onError() {
+        // setIsDeleting(false)
+      }
+    }
+  )
+
+  const handleDelete = async (id: string) => {
+    // setIsDeleting(true)
+    updateGithubProvidedRepositoryActive({
+      id,
+      active: false
+    })
+  }
+
+  return (
+    <Table className="mt-4">
+      <TableHeader>
+        <TableRow>
+          <TableHead className="w-[25%]">Repository name</TableHead>
```

Comment:
```suggestion
          <TableHead className="w-[25%]">Name</TableHead>
```

Be consistent with Git Providers